package.json

```diff
@@ -29,6 +29,7 @@
     "flow": "flow",
```

Comment:
@amgleitman it's not included in the diff, but could you also add "macos" in "files" property (line 19)?
Without it, the npm package will publish without the "macos" folder

Reply:
Done.